### DISCUSSION TITLE
Channel members capabilities

### DISCUSSION THREAD

**Discussion by EtcGonza** [GetStream/stream-chat-flutter#868] [Q&A] [answered]
Hi! I need some help because I trying that all the members of a channel can update the name of a channel. When I create a channel I can pass in the 'extraData' and array of capabilities with the name 'own_capabilities'. But this, at least of what I saw, doesn't represent the capabilities of the others members of the channel. Because the person who creates this channel can change the name of channel, but the rest of the members can't.

**This is the error I get:**
![image](https://user-images.githubusercontent.com/44512291/150174720-97fa63eb-9a67-4e31-9580-8683aa64a767.png)

There is another data to pass when I create a channel to fix this? Or maybe to make all the members like Moderator/Administrator?

Thank you!

**Comment by imtoori** [comment, marked as answer]
hey I can suggest you to take a look at this section of the documentation https://getstream.io/chat/docs/flutter-dart/user_permissions/
Regarding the field `own_capabilities`, we're implementing it in the new v4 of the sdk
